pkg/controller/zookeepercluster/zookeepercluster_controller.go

```diff
@@ -140,6 +139,7 @@ func (r *ReconcileZookeeperCluster) Reconcile(request reconcile.Request) (reconc
 		// Error reading the object - requeue the request.
 		return reconcile.Result{}, err
 	}
+	instance.Status.Init()
```

Comment:
Can we move Init to reconcileClusterStatus?

Reply:
I have moved it.